Makefile

```diff
@@ -362,21 +363,21 @@ test-sind-shell: test-img
 # Code Hygiene targets
 #
 
-validate: ## runs lint checker on sysbox source code and tests
-validate: test-img
+lint: ## runs lint checker on sysbox source code and tests
+lint: test-img
 	@printf "\n** Building sysbox **\n\n"
-	$(DOCKER_SYSBOX_BLD) /bin/bash -c "make validate-local"
+	$(DOCKER_SYSBOX_BLD) /bin/bash -c "make lint-local"
 
-validate-local: validate-sysbox-local validate-tests-local
+lint-local: lint-sysbox-local lint-tests-local
 
-validate-sysbox-local:
-	@cd $(SYSRUNC_DIR) && make validate
-	@cd $(SYSFS_DIR) && make validate
-	@cd $(SYSMGR_DIR) && make validate
-	@cd $(SYSLIBS_DIR) && make validate
-	@cd $(SYSIPC_DIR) && make validate
+lint-sysbox-local:
+	@cd $(SYSRUNC_DIR) && make lint
+	@cd $(SYSMGR_DIR) && make lint
+	@cd $(SYSFS_DIR) && make lint
+	@cd $(SYSIPC_DIR) && make lint
+	@cd $(SYSLIBS_DIR) && make lint
 
-validate-tests-local:
+lint-tests-local:
```

Comment:
Let's change TEST_SCR so that we can cover all scripts within tests/ that don't terminate with ".sh" or ".bash" suffix. You can just grep by the first line of the file ("#!/bin/bash) to determine if it's a shell-based file.

Reply:
Done, thanks!